Samples/Client/Client_Subscribe_Samples.cs

```diff
@@ -177,4 +177,60 @@ public static async Task Subscribe_Topic()
             response.DumpToConsole();
         }
     }
-}
+
+    public static void ConcurrentProcessingWithLimit(CancellationToken shutdownToken, IMqttClient mqttClient)
```

Comment:
Please make sure that the sample can be executed in the console app. This requires that the method has no parameters. You may use hivemqtt or mosquitto public broker in this sample.